org.kde.spectacle.json

```diff
@@ -0,0 +1,74 @@
+{
+  "id": "org.kde.spectacle",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "spectacle",
+  "rename-icon": "spectacle",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "kColorPicker",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DBUILD_EXAMPLE=OFF",
+        "-DBUILD_TESTS=OFF"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/ksnip/kColorPicker.git",
+          "tag": "v0.1.6"
+        }
+      ]
+    },
+    {
+      "name": "kImageAnnotator",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DBUILD_EXAMPLE=OFF",
+        "-DBUILD_TESTS=OFF"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/ksnip/kImageAnnotator.git",
+          "tag": "v0.5.3"
+        }
+      ]
+    },
+    {
+      "name": "spectacle",
+      "buildsystem": "cmake-ninja",
+      "post-install": [
+        "install -Dm644 org.kde.spectacle.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/org.kde.spectacle.svg"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/release-service/22.04.1/src/spectacle-22.04.1.tar.xz",
+          "sha256": "335bc9ab70966d23bdd09614c373d73cf0d8ee09788b22637d5717919e443884",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 8763,
+            "stable-only": true,
+            "url-template": "https://download.kde.org/stable/release-service/$version/src/spectacle-$version.tar.xz"
+          }
+        },
+        {
+          "type": "patch",
+          "path": "oars.patch"
+        },
+        {
+          "type": "file",
+          "url": "https://apps.kde.org/app-icons/org.kde.spectacle.svg",
```

Comment:
why not use the icon from icons/sc-apps-spectacle.svgz ?

Reply:
It is compressed, which the Flatpak scripts seem to ignore.

Reply:
Actually why do you even need to specify this? We don't have it in most of the other apps. What's the problem without this line?

Reply:
https://flathub.org/builds/#/builders/10/builds/11103 This check fails.

Reply:
Also, the default icon is very strange. It seems that the one I am used to on @openSUSE stems from an icon theme.

Reply:
I think i would prefer adding that 128x128 icon to the upstream repositry, and then patching it in instead of fetching the icon from the interwebs.

Can you work on that?

Reply:
https://invent.kde.org/graphics/spectacle/-/merge_requests/144 needs to get squashed, though. Tell me if GitLab can't do it. I used the Web IDE.